lib/screens/main_menu.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import 'package:treasure_of_the_high_seas/screens/play.dart';
+
+class MainMenuPage extends StatelessWidget {
+  MainMenuPage({Key key}) : super(key: key); 
```

Comment:
Do we need this constructor? Seems odd to have the optional param when we're not using it?